SConstruct

```diff
@@ -209,16 +210,25 @@ for path in module_search_paths:
 
 # Add module options.
 for name, path in modules_detected.items():
-    enabled = True
-    sys.path.insert(0, path)
-    import config
-
     try:
-        enabled = config.is_enabled()
-    except AttributeError:
-        pass
-    sys.path.remove(path)
-    sys.modules.pop("config")
+        if env_base["module_" + name + "_enabled"]:
```

Comment:
`env_base` won't have those modules defined in the first place, so this will always throw. `opts.Update(env_base)` is used to update the environment from options, but again module options are not even populated yet.